search-ui/src/requests/search-api.ts

```diff
@@ -20,7 +20,7 @@ export async function getAutoComplete(searchValue: string): Promise<any> {
       return data
     }).catch(error => {
       return error
```

Comment:
can you return a SuggestionResponseI object here like:
```
{
error: { some ErrorI obj }
}
```
and where this is called check for error / emit the error up to the app level for error handling?